packages/now-next/src/index.ts

```diff
@@ -266,6 +267,9 @@ export const build = async ({
     const packageJson = normalizePackageJson(pkg);
     debug('Normalized package.json result: ', packageJson);
     await writePackageJson(entryPath, packageJson);
+  } else if (pkg.scripts && pkg.scripts['build']) {
```

Comment:
This logic is backwards compared to `@now/static-build`.

The `now-build` script should take precedence over the `build` script.

Perhaps the [`getCommand(pkg, 'build')`](https://github.com/zeit/now/blob/f4fe8be4dfe56a144be309b46730e3eb16fa64f9/packages/now-static-build/src/index.ts#L72) function should be moved from `@now/static-build` to `@now/build-utils` so it can be shared between both of these builders.

Reply:
@styfle Correct me if I'm wrong.

In `@now/static-build` only run `now-build` if users provided.

Given `package.json`:
```
{
  "dependencies": {
    "next": "latest",
    "react": "latest",
    "react-dom": "latest"
  }
}
```
should run `now-build`

---

Given `package.json`:
```
{
  "scripts": {
    "build": "..."
  },
  "dependencies": {
    "next": "latest",
    "react": "latest",
    "react-dom": "latest"
  }
}
```
should run `build`

---

Given `package.json`:
```
{
  "scripts": {
    "build": "...",
    "now-build": "..."
  },
  "dependencies": {
    "next": "latest",
    "react": "latest",
    "react-dom": "latest"
  }
}
```
should run `now-build`

> The `now-build` script should take precedence over the `build` script.

I updated this, to make sure `now-build` take precedence 🙇

Let me know if there are more scenarios that I should cover 🙌